documentation/app/components/list-item/index.hbs

```diff
@@ -0,0 +1,212 @@
+      <A.Dropdown as |dd|>
+        <dd.ToggleIcon
+          @icon='more-horizontal'
+          @text='Menu'
+          @hasChevron={{false}}
+        />
+        <dd.Interactive
+          @icon='edit'
+          @text='Edit'
+        />
+      </A.Dropdown>
+    </L.Action>
+  </Cut::ListItem>
+</div>
+<h3>Button</h3>
+<pre><code>
+    &lt;Cut::ListItem as |L|&gt;
+      &lt;L.Content>Service a&lt;/L.Content&gt;
+      &lt;L.Action as |A|&gt;
+        &lt;A.Button @text='Delete' @color="critical" @icon="delete"/&gt;
+      &lt;/L.Action&gt;
+    &lt;/Cut::ListItem&gt;
+  </code></pre>
+<div class="list-container">
+  <Cut::ListItem as |L|>
+    <L.Content>Service a</L.Content>
+    <L.Action as |A|>
+      <A.Button @text='Invalidate'></A.Button>
+    </L.Action>
+  </Cut::ListItem>
+  <Cut::ListItem as |L|>
+    <L.Content>Service b</L.Content>
+    <L.Action as |A|>
+      <A.Button @text='Delete' @color="critical" @icon="delete"></A.Button>
+    </L.Action>
+  </Cut::ListItem>
+</div>
+<h3>Generic</h3>
+<pre><code>
+    &lt;Cut::ListItem as |L|&gt;
+      &lt;L.Content>Service a&lt;/L.Content&gt;
+      &lt;L.Action as |A|&gt;
+        &lt;A.Generic&gt;
+          &lt;Hds::Badge @text="2 permissions" @color='warning' @size="small"/&gt;
+          &lt;@text="3 errors" @type="outlined" @color='critical' @size="small"/&gt;
+          &lt;@text="Terraform" @icon="terraform" @size="small"/&gt;
+        &lt;/A.Generic&gt;
+      &lt;/L.Action&gt;
+    &lt;/Cut::ListItem&gt;
+  </code></pre>
+<div class="list-container">
+  <Cut::ListItem as |L|>
+    <L.Content>Service a</L.Content>
+    <L.Action as |A|>
+      <A.Generic>
+        <Hds::Badge @text="2 permissions" @color='warning' @size="small" />
+        <Hds::Badge @text="3 errors" @type="outlined" @color='critical' @size="small" />
+        <Hds::Badge @text="Terraform" @icon="terraform" @size="small" />
+      </A.Generic>
+    </L.Action>
+  </Cut::ListItem>
+  <Cut::ListItem as |L|>
+    <L.Content>Service b</L.Content>
+    <L.Action as |A|>
+      <A.Generic>
+        <Hds::ButtonSet>
+          <Hds::Button @text="Invalidate" @color="critical" @size="small" />
+          <Hds::Button @text="Use" @color="secondary" @size="small" />
+        </Hds::ButtonSet>
+      </A.Generic>
+    </L.Action>
+  </Cut::ListItem>
+  <Cut::ListItem {{on 'click' this.onItemClick}} as |L|>
+    <L.Content>Service c</L.Content>
+    <L.Action as |A|>
+      <A.Generic>
+        <Hds::Link::Standalone @href="#" @text="2 Permissions" @icon="docs-link" />
+      </A.Generic>
+    </L.Action>
+  </Cut::ListItem>
+  <Cut::ListItem as |L|>
+    <L.Content>Service d</L.Content>
+    <L.Action as |A|>
+      <A.Generic>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris erat nulla, porttitor sed feugiat in, blandit
+          sit amet massa. Nulla fermentum vel erat sit amet convallis. Duis sit amet mi.</p>
+      </A.Generic>
+    </L.Action>
+  </Cut::ListItem>
+</div>
```

Comment:
I'll leave it as of now in case we will still have to test list items for a specific instances, and part of the work for that is already added. 
But please feel free to change routes and anything you think has to be changed. I just want to keep `component/list-item`. 
When we will be adding documentation for List item, I'll update that with the right style.